columnphysics/icepack_tracers.F90

```diff
@@ -1261,7 +1264,18 @@ subroutine icepack_compute_tracers (ntrcr,     trcr_depend,    &
                trcrn(it) = atrcrn(it) / aicen
             else
                trcrn(it) = c0
-               if (it == nt_Tsfc) trcrn(it) = Tocnfrz  ! surface temperature
+               if (it == nt_Tsfc) then
+! tcraig, these old options should be deprecated
+! exist for bit-for-bit backwards compatibility in testing
+                  if (tfrz_option == "mushy_old" .or. &
```

Comment:
This needs to be rectified between the codes.  It looks like they're only there to maintain BFB in Consortium tests.  Is it possible to do this merge in 2 PRs, first BFB and then non-BFB?